src/pages/ProductList/MusicalCard.js

```diff
@@ -2,44 +2,67 @@ import React from 'react';
 import styled from 'styled-components';
 
 const MusicalCard = ({ item }) => {
-  const bookingRate =
-    (item.booking_rating_name / item.booking_rating_total) * 100;
-
   const ratedAge = age => {
     const ageRate = {
-      4: '18',
-      3: '15',
-      2: '12',
+      18: '18',
+      15: '15',
+      12: '12',
     };
     return ageRate[age] || 'All';
   };
 
+  const compareDate = day => {
+    const nowTime = new Date();
+    const nowMonth = nowTime.getMonth() + 1;
+    const nowDate = nowTime.getDate();
+    const musicalDate = Number(day.substring(8, 10));
+    const musicalMonth = Number(day.substring(5, 7));
+    let Dday = 0;
+
+    if (nowMonth < musicalMonth) {
+      if (nowMonth === 2) {
+        Dday = 28 - nowDate + musicalDate;
+      } else if (nowMonth === 4 || 6 || 9 || 11) {
+        Dday = 30 - nowDate + musicalDate;
+      } else {
+        Dday = 31 - nowDate + musicalDate;
+      }
+    } else {
+      Dday = musicalDate - nowDate;
+    }
+    if (Dday > 0) {
+      return 'D-' + Dday;
+    } else {
```

Comment:
```suggestion
        dDay = 28 - nowDate + musicalDate;
      } else if (nowMonth === 4 || 6 || 9 || 11) {
        dDay = 30 - nowDate + musicalDate;
      } else {
       dDay = 31 - nowDate + musicalDate;
      }
    } else {
      dDay= musicalDate - nowDate;
    }
    if (dDay > 0) {
      return 'D-' + dDay;
    } else {
```

대소문자 구분 잘 해 주시는게 좋습니다~